MEMBERSHIP.md

```diff
@@ -5,8 +5,23 @@ to become a member. At the moment, the only privilege membership brings is being
 to vote for the project leader. And, of course, the glory of having your name appended
 to this file.
 
+## Applying
+
 To apply for membership, send a pull request to this file, adding your name and
-GitHub ID. 
+GitHub ID.
+
```

Comment:
Possibly GitHub username instead of ID? There actually is a (numerical) GitHub id, for example see it here for me https://api.github.com/users/vsoch. Is that the id that you want/need?

Reply:
It should be username then. :]